<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>RoBank - Transfers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>22.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufiles">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>files</string>
    </property>
    <addaction name="btnRegTransfers"/>
    <addaction name="btnReportTransfers"/>
    <addaction name="btnHistoryTrans"/>
   </widget>
   <widget class="QMenu" name="menuCredits">
    <property name="title">
     <string>Credits</string>
    </property>
    <addaction name="actionDevelopers"/>
   </widget>
   <addaction name="menufiles"/>
   <addaction name="menuCredits"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDevelopers">
   <property name="text">
    <string>Developers</string>
   </property>
  </action>
  <action name="btnRegTransfers">
   <property name="text">
    <string>Trasnfers register</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="btnReportTransfers">
   <property name="text">
    <string>Report transfers</string>
   </property>
  </action>
  <action name="btnHistoryTrans">
   <property name="text">
    <string>Transfers history</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
